AGENT_INTEGRATIONS.md

```diff
@@ -545,7 +677,6 @@
 * datadog-go-expvar: 1.4.0
 * datadog-gunicorn: 1.7.1
 * datadog-haproxy: 2.2.0
-* datadog-harbor: 1.0.0
```

Comment:
This is an error in `agent_requirements.in` as of `6.13.0`: the integration existed but it was not listed. I assume this meant Harbor was not actually shipped in `6.13.0`, was it? Which means it might be okay to remove it from the changelog.

Reply:
Seems Harbor is present in `6.13.0`:

```bash
$ docker run datadog/agent:6.13.0 /opt/datadog-agent/embedded/bin/pip freeze | grep harbor
datadog-harbor==1.0.0
```

Reply:
Ah right, and also `requirements-agent-release.txt` isn't really used to decide which integrations get shipped, but only prevent downgrades:

https://github.com/DataDog/datadog-agent/blob/4f510bbe0676fb9471d393a11bb0faf3165f8626/omnibus/config/software/datadog-agent-integrations-py3.rb#L277

Will revert this change!